build_tools/ci/cpu_comparison/run_test.py

```diff
@@ -509,8 +516,13 @@ def run_all(
     parser.add_argument(
         "vitis_dir", nargs="?", default="/opt/Xilinx/Vitis/2024.1", type=abs_path
     )
+
+    # TODO(max) how do these get turned off?, how do I run with no verbose? 
     parser.add_argument("--return-on-fail", action="store_true", default=True)
     parser.add_argument("-v", "--verbose", action="store_true", default=True)
```

Comment:
d'oh - yea there's no way to turn these off. the only way to make optional flags that default to true actually work is `action="store_false"` (in which case they should be renamed). I guess drop the default but then add the explicit use of the flag in `ci.yml`?